### DISCUSSION TITLE
days difference between 2 dates

### DISCUSSION THREAD

**Discussion by bridei** [Budibase/budibase#4503] [Help]
Hello
Is there any option to extract the difference in days between 2 dates?
Thanks

**Comment by mjashanks** [comment]
We don't have an in-built function for this, but you can do this using Javascript... either
1. As a formula field on a table
2. As a binding in the `Design` section

A quick google.... https://stackoverflow.com/questions/542938/how-to-calculate-number-of-days-between-two-dates
